wagtailio/project_styleguide/templates/patterns/pages/showcase_page/showcase_page_fragment.html

```diff
@@ -0,0 +1,86 @@
+                    <picture>
+                        {% image item.image max-600x400 format-webp as webp_image %}
+                        <source srcset="{{ image_webp.url }}"
+                                type="image/webp"
+                                alt="{% firstof item.alt_text item.image.alt %}"/>
+                        {% image item.image max-600x400 as image_png %}
+                        <source srcset="{{ image_png.url }}"
+                                type="image/png"
+                                alt="{% firstof item.alt_text item.image.alt %}"/>
+                        {% image item.image max-600x400 as base_img %}
+                        <img
+                            src="{{ base_img.url }}"
+                            height="{{ base_img.height }}"
+                            width="{{ base_img.width }}"
+                            class="showcase-card__image"
+                            alt="{% firstof item.alt_text item.image.alt %}"/>
+                    </picture>
+                    {% if item.logo %}
+                        <picture>
+                            {% image item.logo max-120x100 format-webp as webp_image %}
+                            <source srcset="{{ image_webp.url }}"
+                                    type="image/webp"
+                                    alt="{% firstof item.alt_text item.image.alt %}"/>
+                            {% image item.logo max-120x100 as image_png %}
+                            <source srcset="{{ image_png.url }}"
+                                    type="image/png"
+                                    alt="{% firstof item.alt_text item.image.alt %}"/>
+                            {% image item.logo max-120x100 as base_img %}
+                            <img
+                                src="{{ base_img.url }}"
+                                height="{{ base_img.height }}"
+                                width="{{ base_img.width }}"
+                                class="showcase-card__logo"
+                                alt="{% firstof item.alt_text item.image.alt %}"/>
+                        </picture>
+                    {% endif %}
+                {% endif %}
+                <div class="showcase-card__content">
+                    <h2 class="showcase-card__heading">{{ item.title }}</h2>
+                    <p class="showcase-card__description">{{ item.description }}</p>
+                    <a class="showcase-card__link" href="{% if item.external_link %}{{ item.external_link }}{% else %}{% pageurl item.internal_link %}{% endif %}">{{ item.link_text }} <svg class="showcase-card__svg" width="15" height="16" aria-hidden="true">
+                        <use xlink:href="#{% if self.external_link %}open-link{% else %}arrow{% endif %}"></use>
+                    </svg></a>
+                </div>
+            </div>
+        {% endfor %}
+    </div>
+    <div>
+        <div class="pagination pagination--showcase">
```

Comment:
question: why not hook the pagination to HTMX too?

Reply:
Yep, can do!